projects/shared-modules/hms-video/components/conference-v2/conference-settings/conference-settings.component.ts

```diff
@@ -123,6 +116,11 @@ export class ConferenceSettingsComponent implements OnInit, OnDestroy {
   }
 
   close(value: boolean): void {
+    this.localMediaV2Service.setAudioInputDeviceId(this.selectedAudioInputDeviceId);
+    this.localMediaV2Service.setVideoDeviceId(this.selectedVideoDeviceId);
+    this.localMediaV2Service.setIsAudioEnabled(this.isAudioEnabled);
+    this.localMediaV2Service.setIsVideoEnabled(this.isVideoEnabled);
```

Comment:
Should we set it on close or just directly update it when the device is changed from the dropdown or is muted/unmuted

Reply:
If the user is on the stage with video muted and then opens settings to change their video device and checks if it's working, now if we directly update the service then in the room also their video becomes visible, so that's why I'm updating it when we are closing the settings component